Documentation/contributor-guide/community-membership.md

```diff
@@ -112,8 +112,15 @@ depend on to make decisions in the best interest of the project in a consistent
     - Initiating, contributing and resolving discussions (emails, GitHub issues, meetings)
     - Identifying subtle or complex issues in designs and implementation PRs
 - Directly contributed to the project through implementation and / or review
-- Subscribed to [etcd-maintainers@googlegroups.com]
-- Elected by supermajority of active maintainers.
+- Elected by supermajority of active maintainers, election steps are like below:
+  - Ask two maintainers to become a sponsor
+  - Send an email with candidacy to [etcd-maintainers@googlegroups.com] asking for a vote
+  - Existing maintainers vote privately
+  - After votes are cast one of maintainers responds with result to the original email, either positive or negative with feedback for suggested improvement
+- Open a PR and add an entry to the [MAINTAINERS] file
+- Request to join to etcd-maintainer teams of [etcd organization of GitHub](https://github.com/orgs/etcd-io/teams)
+- Request to join to the private slack channel for etcd maintainers on [kubernetes slack](http://slack.kubernetes.io/)
+- Subscribed to [etcd-maintainers@googlegroups.com] and [etcd-maintainers-private@googlegroups.com]
```

Comment:
Would also like to add:
* Requesting to join [maintainers GitHub team](https://github.com/orgs/etcd-io/teams/maintainers-etcd) or one specific for the project.
* Access to shared passwords <- we should setup a LastPass,1Password or Keybase as provided by CNCF https://www.cncf.io/services-for-projects/#tools
* Access to etcd Google Cloud projects hosting etcd images